MapAdjuster/Source/PointMapper/PointMappingRule.h

```diff
@@ -0,0 +1,25 @@
+#ifndef POINT_MAPPING_RULE_H
+#define POINT_MAPPING_RULE_H
+
+#include "Parser.h"
+
+class PointMappingRule: commonItems::parser
+{
+  public:
+	PointMappingRule() = default;
+	explicit PointMappingRule(std::istream& theStream);
+
+	[[nodiscard]] const auto& getName() const { return name; }
+	[[nodiscard]] const auto& getSource() const { return source; }
+	[[nodiscard]] const auto& getTarget() const { return target; }
+
+  private:
+	void registerKeys();
+
+	std::string name;
+	std::vector<int> source;
```

Comment:
From the test file, these are intended to be coordinates, right? In that case for code clarity, it's worth the effort to create a small class to hold the x/y values and be explicit about them.